user/languages/julia.md

```diff
@@ -92,7 +94,7 @@ to manually install any dependencies that are not handled by `deps/build.jl`,
 it is possible to specify a custom dependency installation command as described
 in the [general build configuration](/user/customizing-the-build/) guide.
 
-In a rare case, you may need to clone a private repo if it is a depenedancy of the repo you are trying to test. To add a private repo, check out the link here: [Private Dependencies](/user/private-dependencies/).  Once you have the repo added, you will need to copy it to your julia folder and then run the default build script.  Check out the script below to see how that is done: 
+In a rare case, you may need to clone a private repo if it is a depenedancy of the repo you are trying to test. To add a private repo, check out the link here: [Private Dependencies](/user/private-dependencies/).  Once you have the repo added, you will need to copy it to your julia folder and then run the default build script.  Check out the script below for __Linux__ to see how that is done: 
```

Comment:
```suggestion
In a rare case, you may need to clone a private repo, if you are trying to test a dependency of the repo. To add a private repo, check out the link here: [Private Dependencies](/user/private-dependencies/).  Once you have the repo added, you will need to copy it to your julia folder and then run the default build script.  Check out the script below for __Linux__ to see how that is done: 
```

Reply:
@Pezi777 In this example, you are not testing a dependancy. You are testing the repo!

Reply:
Therefore I stand by the original language I used.

Reply:
@Pezi777 any thoughts?

Reply:
we can keep the wording, but it is nevertheless "dependency" and not "depenedancy" - right?

Reply:
That’s correct! My mistake on the spelling error.